code/modules/mob/living/carbon/xenomorph/castes/queen/abilities_queen.dm

```diff
@@ -534,51 +534,15 @@
 // ***************************************
 // *********** Queen heal
 // ***************************************
-/datum/action/xeno_action/activable/queen_heal
-	name = "Heal Xenomorph"
+/datum/action/xeno_action/activable/psychic_cure/queen
+	name = "Psychic Cure"
 	action_icon_state = "heal_xeno"
-	mechanics_text = "Heals a target Xenomorph"
-	plasma_cost = 150
-	cooldown_timer = 16 SECONDS
+	mechanics_text = "Heal and remove debuffs from a target."
+	cooldown_timer = 1 MINUTES
+	plasma_cost = 200
 	keybind_signal = COMSIG_XENOABILITY_QUEEN_HEAL
```

Comment:
Just removed the stuff already defined on the base

Reply:
done!